x-pack/plugin/old-lucene-versions/src/main/java/org/elasticsearch/xpack/lucene/bwc/codecs/lucene50/BWCLucene50PostingsFormat.java

```diff
@@ -401,73 +108,4 @@ public FieldsProducer fieldsProducer(SegmentReadState state) throws IOException
             }
         }
     }
-
-    /**
-     * Holds all state required for {@link Lucene50PostingsReader} to produce a {@link
-     * org.apache.lucene.index.PostingsEnum} without re-seeking the terms dict.
-     */
-    public static final class IntBlockTermState extends BlockTermState {
```

Comment:
this is public, we can rely on the Lucene original variant where needed (only in tests with the removal of Lucene50PostingsReader)